<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharmPreferences</class>
 <widget class="QDialog" name="CharmPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Charm Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="lbDurationFormat">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Duration format (requires restart)</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>cbDurationFormat</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="lbToolButtonStyle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Buttons show</string>
       </property>
       <property name="buddy">
        <cstring>cbToolButtonStyle</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="lbTimeTrackerFontSize">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Time tracker window font size</string>
       </property>
       <property name="buddy">
        <cstring>cbTimeTrackerFontSize</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="cbDurationFormat">
       <item>
        <property name="text">
         <string>Minutes (hh:mm)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Decimal (h.xx)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QCheckBox" name="cbIdleDetection">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="lbWarnUnuploadedTimesheets">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Enable Bill Lumbergh</string>
       </property>
       <property name="buddy">
        <cstring>cbWarnUnuploadedTimesheets</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QPushButton" name="pbResetPassword">
       <property name="text">
        <string>Reset password</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="cbToolButtonStyle">
       <item>
        <property name="text">
         <string>Icons</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Text</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Text under icon</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Text beside icon</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>System default</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QCheckBox" name="cbEnableCommandInterface">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QCheckBox" name="cbWarnUnuploadedTimesheets">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="lbRequestEventComment">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Comment finished events</string>
       </property>
       <property name="buddy">
        <cstring>cbRequestEventComment</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="lbIdleDetection">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Enable idle detection</string>
       </property>
       <property name="buddy">
        <cstring>cbIdleDetection</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lbCommandInterface">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Enable command interface</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>cbEnableCommandInterface</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QCheckBox" name="cbRequestEventComment">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="lbResetPassword">
       <property name="text">
        <string>Reset password</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="cbTimeTrackerFontSize">
       <item>
        <property name="text">
         <string>Small</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Regular (Application Font)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Large</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbTimeTrackerFontSize</tabstop>
  <tabstop>cbToolButtonStyle</tabstop>
  <tabstop>cbDurationFormat</tabstop>
  <tabstop>cbIdleDetection</tabstop>
  <tabstop>cbWarnUnuploadedTimesheets</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CharmPreferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CharmPreferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
